### ISSUE TITLE
 sincerely ask for finetuning settings for Davis and Kiba datasets

### ISSUE THREAD

**Issue by koalaaaaaaaaa** [chao1224/GraphMVP#16]
Hi, @chao1224 , I'm sorry to disturb you. I'm wondering whether the finetuning settings (hyperparameters) for kiba and davis is still available at present? I met a problem when I finetuning kiba and davis datasets: the loss is difficult to come to a convergence and the metrics is much worse.  
Hope you may help me when you're free~ Best regards!

**Comment by chao1224** [comment]
Hi @koalaaaaaaaaa,

I didn't tune the hyper-parameters for downstream tasks, and the one I used for DTI is [here](https://github.com/chao1224/GraphMVP/blob/main/scripts_regression/run_fine_tuning_model_DTI.sh).

**Comment by koalaaaaaaaaa** [comment]
thanks a lot! ：）
